public/comt_projects.json

```diff
@@ -17,15 +17,15 @@
n of an unstructured-grid three-dimensional coupled hydrodynamic-water quality model, <em>Continental Shelf Research.</em>, <b>72</b>: 163–179.</p><p>Wang, L., Justic, D. (2009) A modeling study of the physical processes affecting the development of seasonal hypoxia over the inner Louisiana-Texas shelf: Circulation and stratification. <em>Continental Shelf Research</em> <b>29</b>: 1464-1476.</p><p><a href='https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2014JC010634' target='_blank'>Yu, L., K. Fennel, and A. Laurent (2015)  A modeling study of physical controls on hypoxia generation in the northern Gulf of Mexico, <em>Journal of Geophysical Research Oceans</em>, <b>120</b>: 5019–5039.</a></p><img class='img-responsive img-thumbnail' src='/images/comt/justicgrid_blue_zcont.gif' /><p class='caption'>Figure 2: Triangular FVCOM model grid.</p><h3>NCOM</h3><p>An implementation of the Navy Coastal Ocean Model (NCOM; Martin 2000) for the northern Gulf of Mexico shelf has been developed by Dong S. Ko at NRL. The model is nested within the Intra-Americas Sea Nowcast/Forecast System (IASNFS) model (Ko et al. 2003, 2008), which represents the Gulf of Mexico and Caribbean basins with 1/24th degree horizontal resolution (~ 6 km) and 41 hybrid vertical layers. The northern Gulf of Mexico shelf model uses a finer resolution (~2 km). In collaboration with EPA partners John Lehrter and Michael Murrell, IASNFS-LCS was coupled to the Gulf Ecology Model (Eldridge and Roelke 2010) and shown to reproduce the observed salinity, temperature, stratification and mean circulation (Lehrter et al., 2013).</p><p>Eldridge, P.M., and D.L. Roelke (2010), Origins and scales of hypoxia on the Louisiana shelf: importance of seasonal plankton dynamics and river nutrients and discharge, <em>Ecological Modelling</em> <b>221</b>:1028-1042.</p><p>Ko, D.S., R.H. Preller, and P.J. Martin (2003), An experimental real-time Intra-Americas Sea Ocean Nowcast/Forecast System for coastal prediction, paper presented at <em>AMS 5<sup>th</sup> Conference on Coastal Atmospheric and Oceanic Prediction and Processes</em>, Seattle, Washington.</p><p>Ko, D.S., P.J. Martin, C.D. Rowley, and R.H. Preller (2008), A real-time coastal ocean prediction experiment for MREA04, <em>Journal of Marine Systems</em>, 4, 17-28.</p><p>Lehrter, J. C., D. S. Ko, M. C. Murrell, J. D. Hagy, B. A. Schaeffer, R. M. Greene, R. W. Gould, and B. Penta (2013), Nutrient distributions, trans- ports, and budgets on the inner margin of a river-dominated continental shelf, <em>Journal of Geophysical Research Oceans</em>, <b>118</b>:, 4822–4838.</p><p>Martin, P.J. (2000), Description of the navy coastal ocean model version 1.0, <em>NRL/FR/7322--00-9962</em>, Naval Research Laboratory, Stennis Space Center, Mississippi.</p><img class='img-responsive img-thumbnail' src='/images/comt/Kogrid_blue_bathy_coarse2.gif' /><p class='caption'>Figure 3: NCOM grid for northern Gulf of Mexico model. Only every 4th gridline is shown.</p>",
```

Comment:
and just like that, @kellydesent takes the lead for longest single line commit @ RPS 🥇